src/styles/blocks/calendar.scss

```
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  width: $calendar-width;
+  padding: 10px;
+
+  &--start-day {
+    @each $day, $day-number in $days-of-week {
+      &--#{$day} :first-child {
+        margin-left: get-offset($day-number, $day-size, $calendar-gap);
+      }
+    }
+  }
+
+  &--month-length {
+    @for $i from 28 through 31 {
+      &--#{$i} :nth-child(n + #{$i + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    @extend %flex-center;
+    @include square($day-size);
+
+    background-color: #eee;
+    border: 1px solid black;
+
+    transition-duration: 0.5s;
+    transition-property: transform, background-color;
```

```suggestion
    transition: background-color, transform 0.5s;
```
I think u can combine these properties